takwimu/templates/takwimu/_includes/search/_popover.html

```diff
@@ -16,4 +32,70 @@
     border-radius: 0;
     font-weight: normal;
 }
-</style>
\ No newline at end of file
+</style>
+<script>
+    // Reset filters on popover
+    $(".popover").find("#reset-filter").click(function(e){
+        var _parent = $(this).attr("data-parent");
+        $("#results-table tr#result").show();
+        $("input[data-parent="+_parent+"]:checkbox:checked").prop("checked", false);
+
+
+        $("#"+_parent+" button .select-item").removeClass("d-none").addClass("d-flex");            
+        $("#"+_parent+" button .filter-badge").removeClass("d-flex").addClass("d-none");
+    });
```

Comment:
Every element with the popover class will be affected by this. This class included - https://github.com/TakwimuAfrica/TAKWIMU/pull/374/files#diff-a50a162bb144b49aaf29732276df3ad1R23

Please change.

Reply:
@DavidLemayian 
Only the popover that extends this component will be affected, not in a global level. but only the elements extending this template